### PR TITLE
Better readme

### PR DESCRIPTION
**Pull request by aizerin**

### CODE REVIEW

File: src/log.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { tap } from 'ramda';
 
 /**
- * Function with side-effect. Logs input to console and returns that input.
+ * Function with side-effect. Logs input to console and returns that input. Should be used only in development.
  *
  * @func
- * @category Function
+ * @category debugging
```
Comment (tommmyy) [inline src/log.js]: first capital

File: src/trace.js [hunk 2]
```diff
@@ -2,10 +2,10 @@ import { curry } from 'ramda';
 
 /**
  * Function with side-effect. Logs input to console and returns that input.
- * Similar to "log" but allows to label logged value.
+ * Similar to "log" but allows to label logged value. Should be used only in development.
  *
  * @func
- * @category Function
+ * @category debugging
```
Comment (tommmyy) [inline src/trace.js]: first capital

File: README.md [hunk 3]
```diff
@@ -10,6 +10,12 @@ Most of the functions have its own tests and examples in the JS Doc.
 
 Feel free to either contribute yourself or submit an issue if there is a bug or you have an idea for a new extension.
 
+# Why to choose ramda-extension over other libraries ?
+
+We are trying to focus on functional point-free code. Our goal is to keep codebase composed only from ramda functions, not from own code. Basically, we are using only `const` and `ES6 modules` from the JavaScript itself.
```
Comment (tommmyy) [inline README.md]: "We focus on" instead of "We are trying"

File: README.md [hunk 4]
```diff
@@ -10,6 +10,12 @@ Most of the functions have its own tests and examples in the JS Doc.
...
+
+We have a few functions with side-effects (like `log` and `trace`) to support better developer's experience in debugging process. It is only for development purpose and it should not be used in production code.
```
Comment (tommmyy) [inline README.md]: i am no english god.. but "like" to "such as"? 👯‍♂️

Comment (aizerin) [reply README.md]: https://english.stackexchange.com/questions/129976/difference-between-such-as-and-like

**Comment by tommmyy** [comment]
good work

**Pull request merged**